### ISSUE TITLE
JSON quiz

### ISSUE THREAD

**Issue by Graham-Broughton** [Ebazhanov/linkedin-skill-assessments-quizzes#5162]
I'm submitting an ISSUE: please check one with "x"

- [ ] New question{'s} 
- [ ] New assessment 
- [ ] Missed questions/answers 
- [ ] Request for new quiz/answers/...
- [ ] Documentation issue or request for ...
- [x ] ... the answer is incorrect
for Q63 it should be [] instead of {}

**Comment by NaNshekhar04** [comment]
@Graham-Broughton You are correct ! **Parentheses {...}** is used to define a single object, while the other [...] is used to define a sequence of either objects, values or lists in **JSON** ! So the correct answer should be option 1 as **Box Brackets [ ]** are used for enclosing arrays!

**Comment by NaNshekhar04** [comment]
Hey @Graham-Broughton you can PR on the issue and commit the changes with the appropriate Correction !

@Hey @Ebazhanov 👋🏻 This issue can be closed now!

**Comment by Ebazhanov** [comment]
thanks let's wait for PR, then we can close it
